Re-Adding Draenei  (and changing a few models

SebastianTanoni opened the pull request:
New things added in this PR for Draenei:


**Miscelaneous**

Added a new goldmine at the Sentinel start location for the Draenei


**Code**

Sentinel old setup changed to selection

Added the faction selection code


**9 new Upgrades** 

1. Aligned Crystal Mechanism
2. Azerothian Husbandry
3. Crystal Shields
4. Endless Energy
5. Improved Crystal Discharge
6. Kalimdor Wilds Acclimatization
7. Restore Mana
8. Naaru's Shield
9. Protector of Kings


**New spells:**

Crystal Discharge (2 levels) (Crystal Spire)
Summon Elekks (on the Elekk Knight)
Restore Mana (Arcane Sanctum)


**Removed things:**

Elekk from the Crystal Spire


**Quest change:**

Desolace was changed to Darkshore in the "The Way Forward" quest


**Things not needed to be reviewed in this PR**

Changed names and models of Dragonmaw, Zandalar and Goblin objects.

YakaryBovine commented:
Not ready for review yet due to failed checks @SebastianTanoni

YakaryBovine commented on src/WarcraftLegacies.Shared/FactionObjectLimits/DraeneiObjectLimitData.cs:
```diff
@@ -47,6 +47,15 @@ public static IEnumerable<ObjectLimit> GetAllObjectLimits()
       yield return new("H09M", 1); //Adal
       yield return new("R078", Unlimited); //Elementalist training
       yield return new("R07C", Unlimited); //Luminarch training
+      yield return new("RD01", Unlimited); //Defender Upgrade
```
Use Constants library, especially for any new entries

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/925910098.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Improve the stun duration for Crystal Discharge from 0.15 and 1 to 0.30 and 2"
```
That is crazy confusing to read!
```suggestion
      "Value": "Improves Crystal Discharge by increasing its stun duration against units to 1 and its stun duration against heroes to 2."
```

YakaryBovine replied:
Use data fields

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/942687314.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Allows Astral Sanctum to use Restore Mana, which restores 240 mana to a target unit anywhere on the map.\r\n\r\n"
```
```suggestion
      "Value": "Allows Astral Sanctum to use Restore Mana, which restores 240 mana to a target unit anywhere on the map."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/942687314.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Allows Astral Sanctum to use Restore Mana, which restores 240 mana to a target unit anywhere on the map.\r\n\r\n"
```
Use data fields

SebastianTanoni replied:
this is triggered

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/959464530.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Increases Arcane Wells\u2019 mana regeneration by 100%.\r\n\r\n"
```
```suggestion
      "Value": "Increases Arcane Wells\u2019 mana regeneration by 100%."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/909132882.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 1835101799,
+      "Type": 3,
+      "Value": "Azerothian Husbandry\r\n\r\n"
```
```suggestion
      "Value": "Azerothian Husbandry"
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/909132882.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Improves Nether Rays by increasing their hit points by 200 and movement speed by 50.\r\n\r\n"
```
```suggestion
      "Value": "Improves Nether Rays by increasing their hit points by 200 and movement speed by 50."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/909132882.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 829453415,
+      "Type": 3,
+      "Value": "Research Azerothian Husbandry\r\n\r\n"
```
```suggestion
      "Value": "Research Azerothian Husbandry"
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/892355666.json:
```diff
@@ -0,0 +1,49 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Allows Elekk Knights to use Protector of Kings, which gives 30 additional armor to your nearby heroes."
```
Use data fields

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/825246802.json:
```diff
@@ -0,0 +1,49 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Changes the armor type of Defenders to Large.\r\n\r\n"
```
```suggestion
      "Value": "Changes the armor type of Defenders to Large."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/808535122.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Allows Vindicators to use Naaru\u2019s Shield, which reduces the damage taken from Piercing attacks to 25%"
```
```suggestion
      "Value": "Allows Vindicators to use Naaru\u2019s Shield, which reduces the damage taken from Piercing attacks to 25%."
```

YakaryBovine replied:
Use data fields

YakaryBovine commented on mapdata/WarcraftLegacies/BuffData/Skin/B0AX.json:
```diff
@@ -18,7 +18,7 @@
       "Id": 1953653094,
       "Type": 3,
-      "Value": "ReplaceableTextures\\PassiveButtons\\PASKingStrideAura.BLP"
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNKingStrideAura.BLP"
```
Nice catch

YakaryBovine commented on mapdata/WarcraftLegacies/AbilityData/Skin/1397572673.json:
```diff
@@ -0,0 +1,59 @@
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Restores 240 mana to a target unit anywhere on the map.\r\n\r\n"
```
```suggestion
      "Value": "Restores 240 mana to a target unit anywhere on the map."
```

YakaryBovine commented on mapdata/WarcraftLegacies/AbilityData/Skin/1397572673.json:
```diff
@@ -0,0 +1,59 @@
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Restores 240 mana to a target unit anywhere on the map.\r\n\r\n"
```
Use data fields

SebastianTanoni replied:
triggered

SebastianTanoni commented:
@YakaryBovine all done

YakaryBovine commented:
Remove Ultravision
![image](https://github.com/user-attachments/assets/7bd6c4e2-2c72-46f8-9d30-33a671b45bea)

Remove Blight Dispel Small
![image](https://github.com/user-attachments/assets/d18aeeee-4c73-4e04-ab3d-afd1036b1e08)

Weird line break here, I think you probably have line breaks in the Acclimization research name
![image](https://github.com/user-attachments/assets/00251514-b0da-4c79-93e1-4ee75a9fcb58)

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/d6a804c2-80c3-4e89-a702-c33c0a349aa1)
Need autocast icon

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/771e884d-db03-4220-85e3-95edff127621)
Need to adjust the region on this I think, I counted 5 creeps on Azuremyst Isle

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/7dcab8dd-2f23-475d-b3a1-e248152e5c96)
Not only usable on Divine Citadel

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/70b08b84-4b20-445f-b02d-35c5c7af99f0)
Only summons one unit so should just say "Reinforcement"

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/2f2ee13b-1d6c-4ed4-bda1-e5c1df21be09)
Available without the research

SebastianTanoni commented:
il get to it

YakaryBovine commented on mapdata/WarcraftLegacies/AbilityData/Skin/1348087873.json:
```diff
@@ -19,7 +19,7 @@
       "Id": 828536161,
       "Type": 3,
-      "Value": "Charge 1 mana per mana point used to a Divine Citadel directly next to the Arcane Well"
+      "Value": "Charge 1 mana per mana point used to a building directly next to the Arcane Well"
```
```suggestion
      "Value": "Transfers mana from the Arcane Well to the target adjacent friendly structure."
```

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/a3d1dce6-b167-4fe6-8c48-7025b9accd82)
Needs more info. Try "Summons a permanent Defender at the position of the target friendly unit anywhere on the map."

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/a253a2bc-8508-47f1-aad9-a35ec6297fc6)
Need to add that Visual ability to the tooltip automation exemption list. Ask on Discord if you don't remember how

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/842024018.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Increases Rangari hit points by 300.\r\n\r\n"
```
```suggestion
      "Value": "Increases Rangari hit points by 300."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/875578450.json:
```diff
@@ -0,0 +1,53 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Increases Polybolos\u2019 attack speed by 25%.\r\n\r\n"
```
```suggestion
      "Value": "Increases Polybolos\u2019 attack speed by 25%."
```

YakaryBovine commented on mapdata/WarcraftLegacies/UpgradeData/Skin/825246802.json:
```diff
@@ -0,0 +1,49 @@
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Changes the armor type of Defenders to Large."
```
```suggestion
      "Value": "Changes the armor type of Defenders to Heavy."
```

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/5629e302-6d12-43ef-8247-7b89fbaddab9)
Needs custom icon, is not Mana Shield

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/d1e47906-1e9e-4f88-be6f-ba21b4660515)
Remove extra line breaks at bottom

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/4f174712-414a-44d8-9e6f-4f0471e6bd64)
Mechanically identical to Mountain Giant's "Hardened" so should be called Hardened

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/9a3353c4-6f19-4c6a-a5cc-ae573a4a0bf3)
"A New Home" quest doesn't mention these units

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/9005b526-ac86-4cfb-a3b9-992525ce6871)
Autocast icon

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/263fbace-69cc-4c9b-b38e-c25aa1994f7b)
Shouldn't be called Rejuvenation. Rejuvenation doesn't restore mana

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/c5d6c485-da96-4b33-88ef-e41f4b7f1699)
Says "Spirit Wolves" initially then "Shadow Wolves" later. I think Shadow is correct

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/0d000199-0a2e-4412-a5a3-5ba7274c5612)
If I open a Portal to Argus, then a Portal to Tempest Keep, the one to Tempest Keep is unusable. I think we fixed this for the Legion version so probably just need to carry that over to this one

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/c20d409e-a938-4829-b1f0-56e6bd185bee)
That second bullet point shouldn't mention creeps. But also, just replace it with requiring the player to complete "A New Home"

YakaryBovine commented:
![image](https://github.com/user-attachments/assets/eb353b87-3272-4bd6-891a-c7a30bba9db8)
Something about this Tempest Keep quest doesn't feel right. Is there meant to be a Keep here? Did I do the Sunfury cleanup wrong?

YakaryBovine reviewed (changes requested):
Okay, I think that's everything!

SebastianTanoni commented:
> ![image](https://private-user-images.githubusercontent.com/8995690/406402936-c20d409e-a938-4829-b1f0-56e6bd185bee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MjQ1ODQsIm5iZiI6MTczNzcyNDI4NCwicGF0aCI6Ii84OTk1NjkwLzQwNjQwMjkzNi1jMjBkNDA5ZS1hOTM4LTQ4MjktYjFmMC01NmU2YmQxODViZWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjRUMTMxMTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjIzYTBmZjBkMGNkZTY3YjI0MTM4YTVhOWE0ZDg1MDAzNDI1Yjk0Y2JiNjMwYmVmZGViZTI5ZjMyYWZjOTc1MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.V1EpRKLsAY4E-4IfL9w1WlWG8s0b-uVMmwD1JrILsVQ) That second bullet point shouldn't mention creeps. You can change that by changing the range parameter on the objective to 0. But also, just replace it with requiring the player to complete "A New Home"

This would not be in the spirit of the quest, the idea is that the player has to hold it. If he loses it during the game, he shoudnt be able to get the crown.

Although yes the creep part should be removed. The objectives of the quest should just be: Hold the 3 control points at the same time

SebastianTanoni commented:
> ![image](https://private-user-images.githubusercontent.com/8995690/406402480-0d000199-0a2e-4412-a5a3-5ba7274c5612.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MjQ1ODQsIm5iZiI6MTczNzcyNDI4NCwicGF0aCI6Ii84OTk1NjkwLzQwNjQwMjQ4MC0wZDAwMDE5OS0wYTJlLTQ0MTItYTVhMy01YmE3Mjc0YzU2MTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjRUMTMxMTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGIxZTU2ZTI1NmQ4MDExYWNmMzQ4MjVlYjQyZTBjOTIzNDQ2N2FmY2M3NjZhZTVhM2ZmYzM4MGExYzg4NDI0MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Lrj9ONVjEn3V3BIRG7KEoIAWxT1qpEWjxMg4eiskH_A) If I open a Portal to Argus, then a Portal to Tempest Keep, the one to Tempest Keep is unusable. I think we fixed this for the Legion version so probably just need to carry that over to this one

I cannot remmeberfor the life of me how we fixed it, il try to scour the discord for the answer

SebastianTanoni commented:
> Nether Ray research requires Flight, but other factions do not require Flight to get the research associated with air units. Recommend making a decision about what is supposed to require Flight, putting it in the Wiki, and following that

il remove the requirement so it is in line with the others

YakaryBovine commented on src/WarcraftLegacies.Source/Quests/Draenei/QuestTriumvirate.cs:
```diff
@@ -17,9 +17,9 @@ public QuestTriumvirate(LegendaryHero velen) : base("Crown of the Triumvirate",
-      AddObjective(new ObjectiveControlPoint(FourCC("n0BL")));
-      AddObjective(new ObjectiveControlPoint(FourCC("n09X")));
+      AddObjective(new ObjectiveControlPoint(FourCC("n0BH"), 0));
+      AddObjective(new ObjectiveControlPoint(FourCC("n0BL"), 0));
```
I don't think all 3 of these need to have 0 range right? Only the one that has no creeps near it, i.e. the Exodar capital.

SebastianTanoni replied:
yes because those cp are not creeping CP, that is a lategame quest. By that time there is not gonna be any creeps around those.

YakaryBovine replied:
That's not a sure thing, particularly for the Argus one. The Legion slot may not be filled, and once you add Forsaken that probability only goes up.
I'd keep it - it keeps all control point objective quests the same, rather than making these two uniquely snipable in niche situations.

YakaryBovine commented:
> > ![image](https://private-user-images.githubusercontent.com/8995690/406403913-eb353b87-3272-4bd6-891a-c7a30bba9db8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MjQ1ODQsIm5iZiI6MTczNzcyNDI4NCwicGF0aCI6Ii84OTk1NjkwLzQwNjQwMzkxMy1lYjM1M2I4Ny0zMjcyLTRiZDYtODkxYS1jN2EzMGJiYTlkYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjRUMTMxMTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTA4ZmE2MmFmYmUzMTgyMjM5NGY2ZDU1ODM0MTM3NzU3ZDhlMjIzMWQ2YWFhNTc1YTFlNGE0OWFjMzQ3YWRmMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.yKzwEQ691lBHC3iiE3WvGIwQTpKwDmAsme5VqHrvZOY) Something about this Tempest Keep quest doesn't feel right. Is there meant to be a Keep here? Did I do the Sunfury cleanup wrong?
> 
> No this is fine, tempest keep is just the "location"

It's pretty lame though aye? Feel like we should adjust the Sunfury cleanup to make that base neutral instead of removing it. Doesn't make sense to have an objective that says "at Tempest Keep" when there is no such thing

SebastianTanoni commented:
> > > ![image](https://private-user-images.githubusercontent.com/8995690/406403913-eb353b87-3272-4bd6-891a-c7a30bba9db8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MjQ1ODQsIm5iZiI6MTczNzcyNDI4NCwicGF0aCI6Ii84OTk1NjkwLzQwNjQwMzkxMy1lYjM1M2I4Ny0zMjcyLTRiZDYtODkxYS1jN2EzMGJiYTlkYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjRUMTMxMTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTA4ZmE2MmFmYmUzMTgyMjM5NGY2ZDU1ODM0MTM3NzU3ZDhlMjIzMWQ2YWFhNTc1YTFlNGE0OWFjMzQ3YWRmMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.yKzwEQ691lBHC3iiE3WvGIwQTpKwDmAsme5VqHrvZOY) Something about this Tempest Keep quest doesn't feel right. Is there meant to be a Keep here? Did I do the Sunfury cleanup wrong?
> > 
> > 
> > No this is fine, tempest keep is just the "location"
> 
> It's pretty lame though aye? Feel like we should adjust the Sunfury cleanup to make that base neutral instead of removing it. Doesn't make sense to have an objective that says "at Tempest Keep" when there is no such thing

ah well sure! We could keep the capital maybe?

YakaryBovine commented:
> > > > ![image](https://private-user-images.githubusercontent.com/8995690/406403913-eb353b87-3272-4bd6-891a-c7a30bba9db8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MjQ1ODQsIm5iZiI6MTczNzcyNDI4NCwicGF0aCI6Ii84OTk1NjkwLzQwNjQwMzkxMy1lYjM1M2I4Ny0zMjcyLTRiZDYtODkxYS1jN2EzMGJiYTlkYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjRUMTMxMTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTA4ZmE2MmFmYmUzMTgyMjM5NGY2ZDU1ODM0MTM3NzU3ZDhlMjIzMWQ2YWFhNTc1YTFlNGE0OWFjMzQ3YWRmMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.yKzwEQ691lBHC3iiE3WvGIwQTpKwDmAsme5VqHrvZOY) Something about this Tempest Keep quest doesn't feel right. Is there meant to be a Keep here? Did I do the Sunfury cleanup wrong?
> > > 
> > > 
> > > No this is fine, tempest keep is just the "location"
> > 
> > 
> > It's pretty lame though aye? Feel like we should adjust the Sunfury cleanup to make that base neutral instead of removing it. Doesn't make sense to have an objective that says "at Tempest Keep" when there is no such thing
> 
> ah well sure! We could keep the capital maybe?

Yeah I reckon, I'll make a ticket.

YakaryBovine commented:
> ![image](https://private-user-images.githubusercontent.com/8995690/406396182-4f174712-414a-44d8-9e6f-4f0471e6bd64.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4MzgzMjcsIm5iZiI6MTczNzgzODAyNywicGF0aCI6Ii84OTk1NjkwLzQwNjM5NjE4Mi00ZjE3NDcxMi00MTRhLTQ0ZDgtOWU2Zi00ZjA0NzFlNmJkNjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjVUMjA0NzA3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzE5YjIzZjg0NjU4YWU1MzhiYjI0ZGIzMDAyZDEwNWFlMjZlMmYwYjIyNThhMjVhZDdiNjhhZjI5ZGZjMzdjMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Uc1qZKCcIP316DJQprxYYteMUAd7ydntoTmEsDUqQ9w) Mechanically identical to Mountain Giant's "Hardened" so should be called Hardened. Same with the Brown Elekk, and the tooltip to summon them

Like I said here, should be called Hardened, not Hardened Skin. I made it just Hardened a few months ago because many of the units with this ability don't have skin

SebastianTanoni commented:
> > ![image](https://private-user-images.githubusercontent.com/8995690/406396182-4f174712-414a-44d8-9e6f-4f0471e6bd64.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4MzgzMjcsIm5iZiI6MTczNzgzODAyNywicGF0aCI6Ii84OTk1NjkwLzQwNjM5NjE4Mi00ZjE3NDcxMi00MTRhLTQ0ZDgtOWU2Zi00ZjA0NzFlNmJkNjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMjVUMjA0NzA3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzE5YjIzZjg0NjU4YWU1MzhiYjI0ZGIzMDAyZDEwNWFlMjZlMmYwYjIyNThhMjVhZDdiNjhhZjI5ZGZjMzdjMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Uc1qZKCcIP316DJQprxYYteMUAd7ydntoTmEsDUqQ9w) Mechanically identical to Mountain Giant's "Hardened" so should be called Hardened. Same with the Brown Elekk, and the tooltip to summon them
> 
> Like I said here, should be called Hardened, not Hardened Skin. I made it just Hardened a few months ago because many of the units with this ability don't have skin

oh, i didnt catch that nuance. ok!
